### ISSUE TITLE
Min-width on table can cause unwanted scrollbars

### ISSUE THREAD

**Issue by lekoala** [olifolkerd/tabulator#3844]
**Describe the bug**
When loading remote data, renderEmptyScroll will set a min width on the table. Loading data afterwards could cause a scrollbar to appear and with to be less than the min width, causing a horizontal scrollbar

**Tabulator Info**
5.2.7

**Working Example**
https://codepen.io/lekoalabe/pen/qBobLKL

**To Reproduce**
Scroll down and see the scrollbar

**Expected behavior**
No horizontal scrollbar

**Additional context**
Possible (ugly) solution https://github.com/olifolkerd/tabulator/pull/3842

**Comment by lekoala** [comment]
@olifolkerd i'm not setting a min width, renderEmptyScroll does due to the fact it loads a remote data source

https://github.com/olifolkerd/tabulator/blob/c3440c7c81afe37d971cf5a3782f4e6758a8e3e8/src/js/core/RowManager.js#L925

i'm not using a flexed element

**Comment by olifolkerd** [comment]
Hey @lekoala 

Sorry im really confused as to what you are trying to report here, your example seems to be working fine, yes the min width is set when the place holder is set, but it should also be cleared when it is removed.

How do i replicate your issue on your example?

Also you example deffo has the the table inside a flexed container:
```css
section {
    flex: 1 1 0%;
}
```

```html
<section class="scroller">
    <div style="">
      <div id="example-table"></div>
    </div>
```

Cheers

Oli :)

**Comment by lekoala** [comment]
Sorry, i thought the issue was obvious, being, the horizontal scrollbar at the bottom due to the extra pixels from the vertical scrollbar which reduce the available width

<img width="1321" alt="image" src="https://user-images.githubusercontent.com/250762/177780580-314c65a5-7cac-4974-b66b-5c195de7cf0b.png">

And indeed sorry about the confusing part with the flex code. It fails in both scenario (with or without flex)

**Comment by olifolkerd** [comment]
Hey @lekoala 

I have pushed a fix for this to the 5.3 branch and will include it in todays release.

Cheers

Oli :)
